Correctly display password protected stacks in the Message Box

runrevelanor opened the pull request:
Display "pwd" instead of the number of script lines for password
protected stacks used as front scripts, back scripts or library
stacks when displayed in lists in the Message Box.

peter-b commented:
@runrevelanor There are several places in the IDE that need to get a suitable count of script lines to place in a button -- at least the message box and the project browser.

Does it make sense to take this opportunity to add a suitable helper to revidelibrary?

runrevelanor committed:
Correctly display password protected stacks in the Message Box

Display "pwd" instead of the number of script lines for password
protected stacks used as front scripts, back scripts or library
stacks when displayed in lists in the Message Box.

Add the revIDEObjectScriptLines function to the revIDELibrary. This
function returns the number of lines in the script of an object or
"pwd" if the object is on a password protected stack.

runrevelanor committed:
Release note

peter-b commented:
@livecode-vulcan review ok a13a0cb

livecode-vulcan commented:
💙 review by @peter-b ok a13a0cb7f98617c4ebded847913da3f1b8014275
